nixos/doc/manual/release-notes/rl-2305.section.md

```diff
@@ -106,3 +106,5 @@ In addition to numerous new and upgraded packages, this release has the followin
 - The `unifi-poller` package and corresponding NixOS module have been renamed to `unpoller` to match upstream.
 
 - The new option `services.tailscale.useRoutingFeatures` controls various settings for using Tailscale features like exit nodes and subnet routers. If you wish to use your machine as an exit node, you can set this setting to `server`, otherwise if you wish to use an exit node you can set this setting to `client`. The strict RPF warning has been removed as the RPF will be loosened automatically based on the value of this setting.
+
+- [Xastir](https://xastir.org/index.php/Main_Page) can now access AX.25 interfaces via the `libax25` package.
```

Comment:
New entries should not be added first or last to prevent merge conflicts. They get sorted before the release.

Reply:
D'oh. Stupidity on my part, it even says this in the comment above the section.